transforms/replace-flash-with-toast/README.md

```diff
@@ -0,0 +1,59 @@
+# replace-flash-with-toast
+
+The `Flash` component has been deprecated in favour of `Toast` component. This codemod converts all `Flash` instances to `Toast`
+
+```diff
+- import Flash from "carbon-react/lib/components/flash";
++ import Toast from "carbon-react/lib/components/toast";
+
+- <Flash message="hello" />
++ <Toast variant="success" isCenter>hello</Toast>
```

Comment:
Does Flash always look like the Toast success variant?

Reply:
Flash has `as` (which is a legacy name for `variant`) prop set to `success` by default and `isCenter` set to true.
Toast has these set to `warning` and `false`. So that's why if either of these two is not provided than I add the prop with a previous default value to the Toast.